packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -241,13 +218,33 @@ module.exports = async (callback: (error?: any) => number) => {
         if (shouldDeployImplementation || isLibrary) {
           const contract = await deployImplementation(contractName, Contract, argv.dry_run)
           if (shouldDeployProxy || isLibrary) {
-            const proxy = await deployProxy(
-              contractName,
-              contract,
-              addresses,
-              initializationData,
-              argv.dry_run
+            // Changes need another proxy/registry repointing
+            const proxy = await deployProxy(contractName, contract, addresses, argv.dry_run)
+
+            const initializeAbi = (contract as any).abi.find(
+              (abi: any) => abi.type === 'function' && abi.name === 'initialize'
             )
+            console.log(`Adding initialization of ${contractName} to proposal`)
+            if (initializeAbi) {
+              const args = initializationData[contractName]
+              const callData = web3.eth.abi.encodeFunctionCall(initializeAbi, args)
+              console.log(
+                `Initializing ${contractName} with: ${args} (encoded as callData: ${callData})`
```

Comment:
nit: this message implies that state changes are happening now rather than being added to the proposal

Reply:
Maybe it's fine since the previous message explicitly states otherwise

Reply:
Yeah that was my intention